flink-end-to-end-tests/flink-end-to-end-tests-common-kafka/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,9 @@
+flink-end-to-end-test-common-kafka
```

Comment:
Are we bundling `kafka-clients` anywhere? (Do we take any part of the jar and put it into ours?)

Reply:
🤯 @rmetzger could you please help us?

Reply:
The notice checker _does not_ use the output from the dependency plugin, but the output of the shade-plugin. If the notice checker complainted, please link a build where this happened.

Note: The _scala suffix checker_ uses the dependency plugin output.

If a given module is not deployed, then you can exclude it from the notice checks in `tools/ci/java-ci-tools/src/main/resources/modules-skipping-deployment.modulelist`.

Reply:
oh snap it does look at the output of the dependency plugin as well.

Reply:
Ah right, this is for cases where we bundle certain things by manually modifying the target/ directory, e.g., for java version-specific dependencies.

Reply:
The notice checker only partially ignores modules that are listed in modules-skipping-deployment.modulelist.

Let me think about it a bit.

Reply:
If you delete the NOTICE file _and_ add the module to `modules-skipping-deployment.modulelist` then the checker should not complain.

Reply:
Thanks @zentol for the solution! I put ```flink-end-to-end-tests-common-kafka``` into ```modules-skipping-deployment.modulelist``` and the compile phase has succeeded 😄